core/src/main/java/com/radixdlt/RunningRadixNode.java

```diff
@@ -135,8 +136,8 @@ public static RunningRadixNode run(UnstartedRadixNode unstartedRadixNode) {
     return new RunningRadixNode(injector);
   }
 
-  public BFTValidatorId self() {
-    return this.injector.getInstance(Key.get(BFTValidatorId.class, Self.class));
+  public SelfValidatorInfo self() {
+    return this.injector.getInstance(Key.get(SelfValidatorInfo.class, Self.class));
```

Comment:
We probably could/should remove the `Self.class` annotation on this? As the clue's in the name now hah

Reply:
yup, makes sense, done